doc/compiler/jitserver/Usage.md

```diff
@@ -20,143 +20,243 @@ OpenJDK Assembly Exception [2].
-#JITServer: compThreadID=0 has successfully compiled com/ibm/jit/JITHelpers.classIsInterfaceFlag()I
-#JITServer: compThreadID=0 found clientSessionData=00007F50BC4F0880 for clientUID=7315883206984276314 seqNo=3 (isCritical=1) (criticalSeqNo=2 lastProcessedCriticalReq=2)
-#JITServer: compThreadID=0 has successfully compiled jdk/internal/misc/Unsafe.arrayBaseOffset(Ljava/lang/Class;)I
-#JITServer: compThreadID=0 found clientSessionData=00007F50BC4F0880 for clientUID=7315883206984276314 seqNo=4 (isCritical=1) (criticalSeqNo=3 lastProcessedCriticalReq=3)
-...
-```
-JITServer prints out information about each new client connections, new streams, successful compilations, etc.
-The same option can be used on the client to log similar information.
+### Hostname/IP Address
 
+On the client, you can specify server's hostname or IP address with
+`-XX:JITServerAddress` option.
 
-### JITServer version
-Note that JITServer prints out version information that is different from `java -version` output.
-To use JITServer, client and server must have matching versions.
-To check client's version pass it `verbose=\{JITServer\}` option as well.
 ```
-$ java -XX:+UseJITServer -Xjit:verbose=\{JITServer\} -version
+$ java -XX:JITServerAddress=example.com
 
-JITServer is currently a technology preview. Its use is not yet supported
-#JITServer: JITServer version: 1.17.0
-#JITServer: JITServer Client Mode. Server address: localhost port: 38400. Connection Timeout 2000ms
-#JITServer: Identifier for current client JVM: 9238414254742342824
-...
-```
+-or-
 
-### JITServer heartbeat
-Server has a statistics thread running in the background, which can periodically print general information
-about the state of the server. Use `-Xjit:statisticsFrequency=<period-in-ms>` option to enable heartbeat logging.
-The output looks like this:
+$ java -XX:JITServerAddress=127.0.0.1
 ```
-$ jitserver -Xjit:statisticsFrequency=3000` # print heartbeat info every 3 seconds
-
-JITServer is currently a technology preview. Its use is not yet supported.
 
-JITServer is ready to accept incoming requests
-#JITServer: Number of clients : 0
-#JITServer: Total compilation threads : 63
-#JITServer: Active compilation threads : 1
-#JITServer: Physical memory available: 12570 MB
-#JITServer: CpuLoad 6% (AvgUsage 1%) JvmCpu 6%
-```
+By default, the server address is set to `localhost`, i.e. server and client are on the same machine.
 
-## Configuration
+### Port
 
-### Hostname
-On the client, you can specify the server hostname or IP.
-```
-$ java -XX:JITServerAddress=example.com
-```
+By default, communication occurs on port `38400`. You can change this by specifying the `-XX:JITServerPort` suboption as follows:
 
-### Port
-By default, communication occurs on port 38400. You can change this by specifying the `-XX:JITServerPort` suboption as follows:
 ```
 $ jitserver -XX:JITServerPort=1234
```

Comment:
Let's add an ampersand at the end of the server command line to place it in the background.